requirements-test-libraries.txt

```diff
@@ -77,13 +77,13 @@ spacy==3.3.1
 srsly==2.4.3
 swagger-spec-validator==2.7.4
 tableauhyperapi==0.0.14946
-thinc==8.1.0
+thinc==8.0.17

```

Comment:
Is this downgrade intentional?

Reply:
It seems to come from "Update tableauhyperapi from 0.0.14946 to 0.0.15145" (66d5e665b253983494fead24289ecea15184cd03) which is otherwise a no-op and should probably be dropped.

Reply:
> and should probably be dropped

Or fixed to update `tableauhyperapi` as suggested by the title.

Reply:
No. And it was supposed to be upgrading `tableauhyperapi`.

Reply:
Ahh, you're ahead of me. Fixed now.